server/chat-plugins/mafia.ts

```diff
@@ -683,191 +683,191 @@ class Mafia extends Rooms.RoomGame {
-			lynch.count++;
-			lynch.trueCount += this.getLynchValue(userid);
-			lynch.lastLynch = Date.now();
-			lynch.dir = 'up';
-			lynch.lynchers.push(userid);
-		}
-		player.lynching = target;
-		const name = player.lynching === 'nolynch' ? 'No Vote' : this.playerTable[player.lynching].name;
+			vote.count++;
+			vote.trueCount += this.getVoteValue(userid);
+			vote.lastVote = Date.now();
+			vote.dir = 'up';
+			vote.voters.push(userid);
+		}
+		player.voting = target;
+		const name = player.voting === 'novote' ? 'No Vote' : this.playerTable[player.voting].name;
 		const targetUser = Users.get(userid);
-		if (previousLynch) {
-			this.sendTimestamp(`${(targetUser ? targetUser.name : userid)} has shifted their vote from ${previousLynch === 'nolynch' ? 'No Vote' : this.playerTable[previousLynch].name} to ${name}`);
+		if (previousVote) {
+			this.sendTimestamp(`${(targetUser ? targetUser.name : userid)} has shifted their vote from ${previousVote === 'novote' ? 'No Vote' : this.playerTable[previousVote].name} to ${name}`);
 		} else {
 			this.sendTimestamp(
 				name === 'No Vote' ?
 					`${(targetUser ? targetUser.name : userid)} has abstained from voting.` :
 					`${(targetUser ? targetUser.name : userid)} has voted ${name}.`
 			);
 		}
-		player.lastLynch = Date.now();
+		player.lastVote = Date.now();
 		this.hasPlurality = null;
-		if (this.getHammerValue(target) <= lynch.trueCount) {
+		if (this.getHammerValue(target) <= vote.trueCount) {
 			// HAMMER
-			this.sendDeclare(`Hammer! ${target === 'nolynch' ? 'Nobody' : Utils.escapeHTML(name)} was voted out!`);
-			this.sendRoom(`|raw|<div class="infobox">${this.lynchBox()}</div>`);
-			if (target !== 'nolynch') this.eliminate(this.playerTable[target], 'kill');
+			this.sendDeclare(`Hammer! ${target === 'novote' ? 'Nobody' : Utils.escapeHTML(name)} was voted out!`);
+			this.sendRoom(`|raw|<div class="infobox">${this.voteBox()}</div>`);
+			if (target !== 'novote') this.eliminate(this.playerTable[target], 'kill');
 			this.night(true);
 			return;
 		}
-		this.updatePlayersLynches();
+		this.updatePlayersVotes();
 	}
 
-	unlynch(userid: ID, force = false) {
+	unvote(userid: ID, force = false) {
 		if (this.phase !== 'day' && !force) return this.sendUser(userid, `|error|You can only vote during the day.`);
 		let player = this.playerTable[userid];
 
-		// autoselflynch blocking doesn't apply to restless spirits
-		if (player && this.forceLynch && !force) {
+		// autoselfvote blocking doesn't apply to restless spirits
+		if (player && this.forceVote && !force) {
 			return this.sendUser(userid, `|error|You can only shift your vote, not unvote.`);
 		}
 
 		if (!player && this.dead[userid] && this.dead[userid].restless) player = this.dead[userid];
-		if (!player?.lynching) return this.sendUser(userid, `|error|You are not voting anyone.`);
-		if (player.lastLynch + 2000 >= Date.now() && !force) {
+		if (!player?.voting) return this.sendUser(userid, `|error|You are not voting anyone.`);
```

Comment:
```suggestion
		if (!player?.voting) return this.sendUser(userid, `|error|You are not voting for anyone.`);
```
I think this flows better.